android/jni/Android.mk

```diff
@@ -28,3 +28,30 @@ LOCAL_SRC_FILES := src/adldata.cpp \
                    src/chips/nuked/nukedopl3.c
 
 include $(BUILD_SHARED_LIBRARY)
+
+include $(CLEAR_VARS)
+LOCAL_MODULE     := ADLMIDIrt
+
+LOCAL_C_INCLUDES := \
+	$(LOCAL_PATH)/src \
+	$(LOCAL_PATH)/include
+
+LOCAL_ARM_MODE   := arm
+LOCAL_CPP_FEATURES := exceptions
+LOCAL_CFLAGS := -DADLMIDI_DISABLE_MIDI_SEQUENCER -DADLMIDI_DISABLE_MUS_SUPPORT -DADLMIDI_DISABLE_XMI_SUPPORT -DDISABLE_EMBEDDED_BANKS
```

Comment:
When you disabling embedded banks, it will not work until you will load WOPL bank on the fly (I.e. by using `adl_openBankFile()` or `adl_openBankData()` calls to set the bank). Just a small note.

Reply:
intended